src/presentation/pages/NoteSettings.vue

```diff
@@ -2,11 +2,19 @@
   <h1>Note settings</h1>
   <div v-if="noteSettings">
     <TextEdit
+      v-model:value="noteSettings.customHostname"
       :name="'customHostname'"
       :title="'Custom Hostname'"
-      :value="noteSettings.customHostname"
       :placeholder="'example: landing.codex.so'"
     />
+    <Checkbox
+      v-if="noteSettings.id"
```

Comment:
what does this "if" mean? what's the use case?

Reply:
removed